test/verify/check-connection

```diff
@@ -407,21 +407,31 @@ G_MESSAGES_DEBUG=all XDG_CONFIG_DIRS=/usr/local %s -p 9999 -a 127.0.0.90 --local
     @skipImage("Kernel does not allow user namespaces", "centos-7", "rhel-7-6", "rhel-7-7", "debian-stable", "debian-testing")
     @skipImage("Added in 184", "rhel-7-6-distropkg")
     def testCockpitDesktop(self):
+        cases = [(['/cockpit/@localhost/system/index.html', 'system', 'system/index', 'system/'],
+                  ['id="system_machine_id"', 'data-action="shutdown"']
+                 ),
+                ]
```

Comment:
Now that we have this, could it also cover a page that doesn't have index.html, e. g. /cockpit/@localhost/playground/react-patterns.html ?